server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -206,6 +206,63 @@ trait EmailQueryMethodContract {
     }
   }
 
+  @Disabled("JAMES-3597 Deleted messages are exposed over JMAP")
```

Comment:
Shouldn't you remove this `Disabled` on your last commit fixing the issue?

Reply:
Crap